.github/workflows/maven.yml

```diff
@@ -0,0 +1,49 @@
+# This job is to test different maven profiles in master branch again Pull Request raised
+# This workflow targets Java with Maven execution
+
+name: TestNG SDK Test workflow for Maven
+
+on:
+  push:
+    branches: [ "sdk" ]
```

Comment:
https://github.com/browserstack/BStackAutomation/blob/1fa3e8eacbba942a8b7a78910b136ab9551679cb/.github/workflows/nw_linter.yml

Can we try something like this and check if it works?